public/css/game.css

```diff
@@ -0,0 +1,141 @@
+.container {
+  width: 100%;
+  margin: 0 !important; }
+
+.game-cards {
+  height: 180px;
+  border: 1px solid black;
+  width: 130px;
+  margin: 50px;
+  background-color: black;
+  border-radius: 5px;
+  color: white; }
+  .game-cards p {
+    font-size: smaller; }
+
+.players-cards {
+  height: 500px; }
+
+.deck-of-black-cards {
+  box-shadow: -2px -2px 0 0  #000000;
```

Comment:
instead of 0px, write 0...so, you should have something like `-2px -2px 0 0 $color-variable`